portfolio/src/main/java/com/google/sps/servlets/DataServlet.java

```diff
@@ -26,11 +26,20 @@
 import javax.servlet.http.HttpServlet;
 import javax.servlet.http.HttpServletRequest;
 import javax.servlet.http.HttpServletResponse;
+import javax.servlet.ServletException;
  
 /** Servlet that adds comments to datastore */
 @WebServlet("/data") 
 public class DataServlet extends HttpServlet 
 { 
+  public void init() throws ServletException {
+    try {
+        languageService = LanguageServiceClient.create();
```

Comment:
I think you have to define this as a private variable in this class.

`private LanguageServiceClient languageService;`

And then you can initialize and access it in your functions.

Reply:
I'll fix that, I believe I sent an older copy of the file, sorry